_posts/2020/2020-03-10-working-as-a-scrum-master-at-epages.md

```diff
@@ -0,0 +1,113 @@
+---
+layout: post
+title: Working as a Scrum Master at ePages
+date: 2020-03-10
+header_image: public/scrum-remote-interview.jpg
+header_overlay: true
+category: on-the-job
+tags: ["scrum", "agile", "job"]
+authors: ["Aroso", "Miriam"]
+about_authors: ["amahaiuddin", "mkofler"]
+---
+
+We've already explained in our previous blog posts how agile working at ePages looks like, e.g. [how Scrum is used at ePages](/blog/methods-and-tools/epages-turns-agile/), [what it's like to be an agile technical writer](/blog/methods-and-tools/what-its-like-to-be-an-agile-technical-writer/), or [how to introduce new employees to Scrum](/blog/methods-and-tools/how-to-introduce-new-employees-to-scrum/).
+But have you ever wondered how the daily work of a Scrum Master looks like and which challenges they face?
+Yes?
+Me too. 🙋🏻‍♀️ 
+That's why I decided to do an interview with one of our Scrum Masters from Jena.
+
+## Moin Miri, thank you for taking the time for this interview. Let's start with your journey to ePages: What's your background and your job position?
+
+Hi, I started in November 2019 as a Scrum Master at ePages.
+In order to tell you how I became a Scrum Master, and how I became an ePagee, I need to go into a little more detail.
+
+I did my master's degree in Intercultural Human Resources Development and Communications Management in Jena.
+The studies included various topics: from digital work to agile work, and new work in combination with psychological concepts.
```

Comment:
What do you mean with "new work"?

Reply:
New Work is a concept for new working methods --> in capital letters

Reply:
Wow... I didn't know that something like this exists :D